### DISCUSSION TITLE
How to rename the file after uploading in Upload component in Ant Design?

### DISCUSSION THREAD

**Discussion by tungvu2210** [ant-design/ant-design#39865] [Q&A]
Due to the name of my file is too long, I would like to rename it to 'xxx...xxx.jpg'. I tried itemRender or beforeUpload but no success.
Thank you for reading!!!
![image](https://user-images.githubusercontent.com/66018884/209778495-66c290c9-1037-4bdd-87cf-d61e27bb4f86.png)

**Comment by bombillazo** [comment]
Wondering the same, how can we rename the file?

**Comment by kaushik-implies** [comment]
same problem @tungvu2210  @bombillazo
